pkg/snet/network.go

```diff
@@ -165,8 +168,16 @@ func New(conf Config, eb *appevent.Broadcaster, masterLogger *logging.MasterLogg
 			SK:              conf.SecKey,
 			AddressResolver: conf.ARClient,
 		}
-
-		clients.Direct[tptypes.SUDPH] = directtp.NewClient(sudphConf, masterLogger)
+		// SUDPH wont be initilised only if the visor is under any symmetric NAT
```

Comment:
Would shorten to just `// sudph will only be initialized if NAT is determined not to be symmetric`

Reply:
Will make the change.